avm/ptn/authorization/role-assignment/tests/e2e/sub.min/main.test.bicep

```diff
@@ -46,9 +57,10 @@ module testDeployment '../../../main.bicep' = {
   name: '${uniqueString(deployment().name)}-test-${serviceShort}'
   params: {
     principalId: nestedDependencies.outputs.managedIdentityPrincipalId
-    roleDefinitionIdOrName: 'Resource Policy Contributor'
+    roleDefinitionIdOrName: '/providers/Microsoft.Authorization/roleDefinitions/c2f4ef07-c644-48eb-af81-4b1b4947fb11'
     principalType: 'ServicePrincipal'
     location: resourceLocation
     subscriptionId: subscriptionId
+    resourceGroupName: resourceGroupName
```

Comment:
Why the resourceGroupName? Does it then not apply the roleAssignment to the RG scope, as opposed to the subscription scope that we want to test?